nixos/modules/services/audio/navidrome.nix

```diff
@@ -28,10 +28,17 @@ in {
         '';
       };
 
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Whether to open the firewall port for NaviDrome";
```

Comment:
NaviDrome -> Navidrome, or the following suggestion:
```suggestion
        description = lib.mdDoc "Whether to open the TCP port in the firewall";
```

Reply:
I *swear* I had looked at that capitalization (which I thought was odd). Thanks!